tests/lambda_endpoints/conftest.py

```diff
@@ -36,38 +36,18 @@ def science_file():
 @pytest.fixture(scope="module")
 def spice_file():
     """Path to a valid spice file."""
-    return "imap/spice/ck/test_v000.bc"
+    return "spice/ck/test_v000.bc"
 
 
 @pytest.fixture(autouse=True, scope="module")
-def s3_client(science_file, spice_file):
+def s3_client():
     """Mock S3 Client, so we don't need network requests."""
     with mock_s3():
         s3_client = boto3.client("s3", region_name="us-east-1")
 
         s3_client.create_bucket(
             Bucket=BUCKET_NAME,
         )
-        result = s3_client.list_buckets()
```

Comment:
Why was this test removed?

Reply:
This is actually a fixture and not a test, so shouldn't be asserting things, the dependent tests will fail if this isn't set up properly.

But, the real reason I removed it was that this was actually creating an s3 bucket **and** populating it with items, and I'm saying we don't want the population of items here (because those items aren't added to the DB properly) and we'll populate it later for each test that needs it.

Reply:
Sounds good, thanks for explaining!z